app/getRealmApp.js

```diff
@@ -3,9 +3,9 @@ import Realm from 'realm';
 let app;
 
 // Returns the shared instance of the Realm app.
-const getRealmApp = () => {
+export const getRealmApp = () => {
   if (!app) {
-    const appId = 'findourdevices-gvqwl'; // Set Realm app ID here.
+    const appId = 'findourdevices-jkqdq'; // Set Realm app ID here.
```

Comment:
Maybe we should put the app id in a config file, or have an `npm` script that configures it, in the interest of more easily setting up the app.